package.json

```diff
@@ -1,84 +1,90 @@
 {
-    "browserslist": {
```

Comment:
Please revert back to 4 spaces, for more relevant diffs and easier comparison to Ibexa version of `package.json`